cmd/proxy/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"log/slog"
+	"os"
+
+	"github.com/absmach/propeller/proxy"
+	"github.com/absmach/propeller/proxy/config"
+	"github.com/caarlos0/env/v11"
+	"golang.org/x/sync/errgroup"
+)
+
+const (
+	svcName  = "proxy"
+	logLevel = "info"
+)
+
+func main() {
+	g, ctx := errgroup.WithContext(context.Background())
+
+	var level slog.Level
+	if err := level.UnmarshalText([]byte(logLevel)); err != nil {
+		log.Fatalf("failed to parse log level: %s", err.Error())
+	}
+	logHandler := slog.NewJSONHandler(os.Stdout, &slog.HandlerOptions{
+		Level: level,
+	})
+	logger := slog.New(logHandler)
+	slog.SetDefault(logger)
+
+	mqttCfg := config.MQTTProxyConfig{}
```

Comment:
There is some stutter here in config MQTT Proxy Config

You can move the configs to the proxy package and import it as 

```go
mqttConfig := proxy.MQTTConfig{}
```